clang/lib/Parse/ParseOpenACC.cpp

```diff
@@ -69,6 +69,29 @@ OpenACCDirectiveKindEx getOpenACCDirectiveKind(Token Tok) {
       .Default(OpenACCDirectiveKindEx::Invalid);
 }
 
+// Translate single-token string representations to the OpenCC Clause Kind.
+OpenACCClauseKind getOpenACCClauseKind(Token Tok) {
```

Comment:
Should this be static (ditto for the rest of the functions in the files)

Reply:
They are instead in an anonymous namespace,which has the same effect with a much less likelihood of someone forgetting static :D

Reply:
I missed that!